logos/obsproject.svg

```diff
@@ -0,0 +1 @@
1.004.121.01.232.006.16.003.07.003.08.013.3.007.163.002.037.002.041.022.48.002.058.029.576.001.032.009.163.016.311.004.072.003.066.01.166.013.236.006.111.005.098.009.146.01.184.01.152.007.128.008.119.008.14.012.196.01.155.006.089.006.098.016.248.012.174.003.058.004.058.023.329.011.164.002.025.04.566.002.017.011.153.026.357.004.047.004.051.014.184.02.254.006.084.007.085.014.169.016.198.01.122.009.118.012.143.013.153.007.079c.026.316.054.631.082.947l.004.043.007.077c.06.665.123 1.329.19 1.992l.006.065.004.034c.032.32.065.639.099.958l.008.073.007.065.104.952.005.046.008.069c.037.33.075.66.114.989l.066.553.059.478.003.031.002.013c.041.329.082.657.125.985l.007.06.006.045.128.969.005.041.008.054.137.997.002.01c.096.689.197 1.377.301 2.064l.005.031.003.021.155 1.001.004.027.005.03c.203 1.286.419 2.568.647 3.846l.054.298c-6.213-21.066-9.547-43.36-9.547-66.428 0-126.487 100.254-229.743 225.567-234.573C396.429 454.371 462.928 500 540 500c11.331 0 22.434-.986 33.227-2.878a234.129 234.129 0 0 1 19.889 38.606C566.77 568.535 551 610.19 551 655.5c0 33.291 8.513 64.609 23.48 91.895C532.993 813.792 459.243 858 375.25 858c-33.767 0-65.878-7.145-94.904-20.004l.839.265.022.007.034.01c.552.173 1.104.344 1.658.512l.035.01.044.014c.547.165 1.095.328 1.643.489l.046.014.05.014c.544.159 1.089.316 1.635.471l.056.015.049.014c.544.154 1.089.305 1.636.454l.062.017.046.012c.273.075.547.148.821.222l.497.132.324.085.065.017.041.011.814.211.024.006.054.014.398.102.366.092.063.016.033.008c.27.068.54.136.811.202l.039.01.07.017.343.085.416.101.055.013.023.005c.27.066.539.13.81.194l.053.013.079.018.296.07.477.111.035.008.012.003.811.186.066.015.081.018c.541.122 1.082.242 1.625.36l.076.016.08.017c.54.117 1.081.231 1.623.343l.086.018.074.015.81.166.012.002.04.008.492.098.274.054.093.019.066.012.805.157.028.005.072.014.421.08.313.058.095.018.057.01.803.148.043.008.09.016.364.066.357.063.093.017.045.007c.267.048.535.094.803.14l.057.01.099.017.316.054.41.068.082.014.033.006.805.132.07.011.101.016.275.044.474.075.06.01.019.003c.27.042.54.084.81.124l.082.013.099.015.237.036.56.083.021.003.817.118.093.013.092.013.817.114.581.078.24.032.102.014.084.01c.27.036.541.071.811.105l.023.003.07.009.465.059.278.034.106.013.074.009c.27.032.539.065.809.096l.038.005.094.011.401.046.319.036.108.013.062.006c.269.031.539.06.809.089l.053.006.106.011.349.037.365.038.104.011.049.005.81.081.067.006.111.011.304.029.422.04.091.009.034.003.814.073.08.007.109.01.266.023.49.041.065.006.019.001c.274.023.547.045.821.066l.091.007.104.008.23.018.584.044.018.001.829.059.101.007.096.006c.275.019.551.037.826.054l.013.001.047.003.53.033.25.015.109.006.086.005.822.046.029.002.084.004.453.024.288.014.113.006.075.003.821.038.044.002.103.004.393.017.332.013.113.005.061.002.822.03.058.002.112.004.343.011.382.012.106.003.048.001c.275.008.549.016.824.022l.073.002.114.003.299.007.442.009.09.002.032.001.829.014.085.001.111.002.262.003.516.006.059.001h.016c.279.003.557.005.836.006l.096.001h.105l.845.002.619-.001z"/></g></g></g></svg>
```

Comment:
Great! Could you provide a more simplified SVG, or a png? I'm worried this will be a bit big for user downloads.

Reply:
Sure, I'll see what I can do.